frontend/src/App.js

```diff
@@ -1,11 +1,38 @@
+import { useState, useEffect } from 'react'
+import getProducts from './services/getProducts';
+import getProductById from './services/getProductById';
 
+export default function App() {
+
+  const [products, setProducts] = useState([])
+  const [productById, setProductById] = useState([])
```

Comment:
You could consider to put the product object with start values here.
Think it would improve the readability. :)

Reply:
good point! I will add it in my next user story